src/components/DropZone/DropZone.scss

```diff
@@ -38,8 +38,19 @@ $dropzone-stacking-order: (
   border: border-width(thick) $dropzone-border-style transparent;
 }
 
+.DropZoneWrapper {
+  height: 100%;
+
+  > div {
```

Comment:
This is always a bit risky to do, because if the `Labelled` wrapping markup changes we need to make sure that this will change as well. I think thanks to the `splash` build tooling that `polaris` uses, we will get a notification that at least indicates `DropZone` will be affected by `Labelled` changes. Still... I'm wondering if we should qualify this with a comment to express why we need to override some `Labelled` styles?

Reply:
Probably a comment or it'll be forgotten about and list in the 💨